### PR TITLE
Has a minimal theme added

### PR DESCRIPTION
**Pull request by markkemper1**
Image below, I blanked out  the pipeline names and  author last name but you can see the overall theme. 
image

### CODE REVIEW

**Comment by markkemper1** [comment]
I've only tested with Buildkite plugin.

File: app/public/stylesheets/themes/minimal/style.css [hunk 1]
```diff
@@ -0,0 +1,161 @@
+#pluginWrap {
+  display: flex;
+  height: 100vh;
+  display: flex;
+  align-items: center;
```
Comment (Alea81) [inline app/public/stylesheets/themes/minimal/style.css]: This should be removed! With a count more than 24 it's slipping out of the window

**Comment by marcells** [comment]
Cool, very long time ago, we added a new theme! :+1:

**Comment by marcells** [comment]
@Alea81 Oh, damn, sorry. I've overseen, that you reviewed it.

@markkemper1 Could you please fix the issue @Alea81 in a new PR?

**Comment by Alea81** [comment]
Have checked because we needed also a "smaller boxed theme".
Never mind, seems o.k. now, was a browser problem I assume...
Thank you for the theme! 🥇

**Pull request merged**